CharlieBackend.Api.UnitTes/CurrentUserTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Moq;
+using Xunit;
+using System;
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Http;
+using CharlieBackend.Core.Entities;
+using CharlieBackend.Business.Services;
+
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class CurrentUserTests : TestBase
+    {
+        [Fact]
+        public void ThrowErrorIfNotLoggined()
+        {
+            var _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            _httpContextAccessorMock.Setup(req => req.HttpContext)
+                .Returns(new DefaultHttpContext());
+            var currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            Assert.Throws<UnauthorizedAccessException>(() => currentuserService.AccountId);
+        }
+
+        [Fact]
+        public void getAccountId()
+        {
+            var _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            var httpContext = new DefaultHttpContext();
+            var mockIdentity = new Mock<GenericIdentity>("Vasya");
```

Comment:
I think you can move this repetitive boilerplate code from each test method into `CurrentUserTests` class constructor, can't you?

Reply:
Partially solved in https://github.com/ita-social-projects/WhatBackend/pull/504/commits/e24ea4bfff6c941a77091f4f006367542332ab6b